src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,8 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  width: 1024px;
```

Comment:
Don't hardcode width, checklist 11th item, use max-width with formula to calculate it